### PR TITLE
Simplify how scripts get icon slugs

### PR DESCRIPTION
**Pull request by ericcornelissen**
**Issue:** n/a
**Alexa rank:** n/a
  

### Checklist
  - [n/a] I updated the JSON data in `_data/simple-icons.json`
  - [n/a] I optimized the icon with SVGO or SVGOMG
  - [n/a] The SVG `viewbox` is `0 0 24 24`

### Description

~~This changes and renames the `titleToSlug` (formerly `titleToFilename`) function once more to avoid having to do `icon.slug || titleToSlug(icon.title)` whenever you're calling the function.~~

~~There are two exceptions to this, one in the `get-filename.js` script where all we have is the brand name; and one in the `build-package.js` script which will be removed in #4920. For this reason we might choose to go with a slightly different approach to refactoring this, namely:~~

- ~~Keep `titleToSlug` in the utilities file.~~
- ~~Add `getIconSlug` and define it as `getIconSlug = icon => icon.slug || titleToSlug(icon.title)`.~~

~~Let me know if that is preferred, I'm happy either way.~~

This adds a utility function called `getIconSlug` that is a basically a shorthand for doing `icon.slug || titleToSlug(icon.title)`.

The changes in `scripts/get-filename.js` are purely for consistency.

### CODE REVIEW

File: scripts/utils.js [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
 module.exports = {
   /**
-   * Converts a brand title into a slug/filename.
-   * @param {String} title The title to convert
+   * Get the slug/filename for an icon.
+   * @param {String} title The icon data as it appears in _data/simple-icons.json
```
Comment (mondeja) [inline scripts/utils.js:9]:
```suggestion
   * @param {Object} icon The icon data as it appears in _data/simple-icons.json
```

**Review by mondeja — CHANGES_REQUESTED** [review]
> - Add `getIconSlug` and define it as `getIconSlug = icon => icon.slug || titleToSlug(icon.title)`.

For consistency, I think that this would be the right choice.

**Comment by ericcornelissen** [comment]
> For consistency, I think that this would be the right choice.

Fair enough, updated the Pull Request :+1:

**Review by mondeja — APPROVED** [review]
Perfect, lets go ahead with the merge :+1:

**Pull request merged**